modules/displayDate.js

```diff
@@ -0,0 +1,11 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export default function displayDate() {
+  const displayDate = document.getElementById('displayDate');
+  const dt = DateTime.now();
+  displayDate.innerHTML = `${dt}`;
+}
```

Comment:
- [ ] [OPTIONAL] I recommend that you consider formatting the date and time on your website in a more user-friendly way, so that non-technical users can easily understand it. Currently, it appears that you are displaying raw date and time strings directly in your HTML, which can be confusing for some users. I suggest using a format like DD-MM-YY HH:MM:SS, or another format that is easy to understand. By doing so, you can improve the user experience on your website.
```suggestion
export default function displayDate() {
  const displayDate = document.getElementById('displayDate');
  const dt = DateTime.now();
  const formattedDate = dt.toFormat('dd-MM-yyyy HH:mm:ss');
  displayDate.innerHTML = formattedDate;
}
```